docs/excel/custom-functions-troubleshooting.md

```diff
@@ -38,6 +38,22 @@ Generally, these errors correspond to the errors you might already be familiar w
 
 Information about custom functions is cached by Office. Sometimes while developing and repeatedly reloading an add-in with custom functions your changes may not appear. You can fix this by clearing the Office cache. For more information, see [Clear the Office cache](../testing/clear-cache.md).
 
+### Clear the custom functions cache when your add-in runs
+
+There may be times when you need to clear the custom functions cache for an add-in deployed to your end users, so that add-in updates and custom functions setting changes are incorporated at the same time. Without triggering a custom functions cache clear, custom functions setting changes may take up to 24 hours to reach your end users.
+
+To ensure that the custom functions cache is cleared by your add-in, add the following code sample to your **functions.js** file.
+
+```javascript
+function setForceRefreshOn() {  
```

Comment:
So is this just automatically called when needed by Office.js? Or do I as the developer need to call this?

Reply:
Good question, thanks! I will confirm with the PM.

Reply:
This method needs to be called in Office.onReady. I added details to the article.

Reply:
Please review again when you have time, thanks @davidchesnut!